templates/components/header.html

```diff
@@ -0,0 +1,44 @@
+<header class="fixed left-0 right-0 top-0 z-10">
+    <div class="flex flex-wrap px-5 md:h-20 justify-between bg-[#202326] items-center text-white">
+        <a class="h-12 md:w-auto my-2 shrink-0 " href="index.html">
+            <img class="h-full" alt="Kivy logo" title="Kivy" src="./static/images/logo_kivy_white.png" />
+        </a>
+        <div class="flex items-center py-2 md:order-last">
+            <a class="px-2" href="https://github.com/kivy">
+                <img class="w-4 md:w-6 invert" src="./static/images/icons/github.svg" />
+            </a>
+            <a class="px-2" href="https://www.youtube.com/user/kivyframework">
+                <img class="w-4 md:w-6 invert" src="./static/images/icons/youtube.svg" />
+            </a>
+            <a class="px-2" href="https://www.facebook.com/kivyframework">
+                <img class="w-4 md:w-6 invert" src="./static/images/icons/facebook.svg" />
+            </a>
+            <a class="px-2" href="https://twitter.com/kivyframework">
+                <img class="w-4 md:w-6 invert" src="./static/images/icons/twitter.svg" />
+            </a>
+            <a class="px-2" href="https://chat.kivy.org/" class="icon-discord">
+                <img class="w-4 md:w-6 invert" src="./static/images/icons/discord.svg" />
```

Comment:
Have we considered leveraging something like FontAwesome instead?
Probably most of the icons are available there and can be easily customized on the fly.

Reply:
The icons located at `/static/images/icons` are all SVG icons from `FontAwesome`.

I preferred to not include the whole `css` and `ttf` files from `FontAwesome` for performance purposes.
(The whole set of icons is quite huge, and Google Pagespeed will complain about it 😂)

The last time I checked there was a tool, that runs on top of FontForge (or something similar) to clean up TTF files from unused icons.

Reply:
Ah yeah I see good point.
When you go with a frontend framework, you can get to have only the ones used bundled.
CDN could also be an option since it's supposed to be cached or custom kit https://fontawesome.com/docs/web/setup/use-kit, but I've never tried.
Anyway I think we're good with what you've setup already and we could still iterate over it later post merge if we see opportunities for improvements